src/middlewared/middlewared/plugins/account.py

```diff
@@ -155,6 +155,20 @@ class Config:
                         'exists and is not a directory',
                         errno.EEXIST
                     )
+
+                # The TLD won't get chowned by walking
+                os.chown(data['home'], data['uid'], group['gid'])
+
+                # The user already created this, to avoid unexpected issues, we
+                # set the user and group as the owner for ALL of the data in the
+                # dataset supplied
```

Comment:
I dont believe we want to chown all the files within, besides that could be a long running operation task and hang everything.

We are reaching for keeping the same behavior of 11.0 which was chown the dir.